dotnet/src/SemanticKernel.Skills/Skills.Memory.VectorDB/QdrantMemoryStore.cs

```diff
@@ -0,0 +1,172 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I think this file is duplicated

Reply:
I think it is. Let me commit fresh to make sure there are not 2 files from copying over.

Reply:
Commit: https://github.com/tawalke/semantic-kernel/pull/1/commits/dd3d73b5fd8f67ab582f720d5190589c70bc82d3 removes duplicate, dated file copied over.